web/next.config.mjs

```diff
@@ -1,88 +1,11 @@
-// @ts-check
```

Comment:
don't change all of this, only remove what's relevant for removing the S3 bucket

Reply:
don't change all of this, only remove what's relevant for removing the S3 bucket